index.html

```diff
@@ -230,7 +245,7 @@ <h4 class="subtitle-color">LET'S CONNECT</h4>
             checked="checked"
           />
           <h2 class="handle">
-            <label for="handle1">Languages <img src="./assets/arrow.png" class="arrow-margin" height="13" width="18" alt="arrow"></label>
+            <label for="handle1">Languages<img src="./assets/arrow.png" class="arrow-margin" height="13" width="18" alt="arrow"></label>
           </h2>
           <div class="content">
 
```

Comment:
- [ ] The language part you should have **JavaScript & HTML & CSS** and also they should be in inline position.

![Capture web_23-9-2021_103651_www figma com](https://user-images.githubusercontent.com/61562626/134485864-a170de4f-63ce-41fa-b4ab-747aa8abd9bc.jpeg)